apps/linkedin-to-notion/src/contents/helpers.ts

```diff
@@ -0,0 +1,8 @@
+import type { LinkedInProfileInformation } from '~src/contents/linkedin-profile-scraper';
+
+export const isScrapingComplete = (values: LinkedInProfileInformation): boolean => {
+  if (!values) {
+    return false;
+  }
+  return !Object.values(values).some((value) => !value);
```

Comment:
This is slightly more elegant I think:

```suggestion
  return Object.values(values).every(value => value);
```